docs/api/app.md

```diff
@@ -647,6 +647,27 @@ Sets the application's [dock menu][dock-menu].
 
 Sets the `image` associated with this dock icon.
 
+### `app.launcher.setBadgeCount(count)` _Linux_
+* `count` Integer
+
+Sets the number to be displayed next to the app icon in the unity launcher.
+Setting count to `0` will hide the badge.
+
+**Note:** This feature is currently only supported on Ubuntu Unity. Calling this function has no effect, when the application is running in a different environment.
+
+**Note:** you need to specify the .desktop file name to the desktopName field in package.json. By default, it will assume app.getName().desktop in packaged apps.
+
+### `app.launcher.getBadgeCount(count)` _Linux_
```

Comment:
This method takes no arguments right? Looks like it takes a `count` right now from the docs.